src/awareness.rs

```diff
@@ -309,6 +309,14 @@ pub struct Subscription<T> {
     subscribers: Weak<RefCell<HashMap<u32, Box<dyn Fn(&Awareness, &T) -> ()>>>>,
 }
 
+impl<T> Into<SubscriptionId> for Subscription<T> {
```

Comment:
The subscriptionID alone won't give you what you want if you don't have companion `unsubscribe` method.

Reply:
Is that something that you would be open to add? I am happy to extend the scope of this PR and add it immediately, I just wonder how we would approach naming. As `observe/unsubscribe` seems to be the names chosen in `y-crdt`, and I wonder what you think is best here?